examples/stable-diffusion/README.md

```diff
@@ -522,4 +522,20 @@ python text_to_image_generation.py \
     --use_habana \
     --use_hpu_graphs \
     --gaudi_config Habana/stable-diffusion
+### Unconditional Image Generation Example
```

Comment:
```suggestion

### Unconditional Image Generation Example
```
You need to add a blank line otherwise it is not rendered properly: https://github.com/cfgfung/optimum-habana/tree/req277/unconditional_image_generation/examples/stable-diffusion#stable-diffusion-xl-inpainting